tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -1703,6 +1703,63 @@ public:
       return CreateAction<RDFInternal::ActionTags::Graph, V1, V2>(validatedColumns, graph, graph);
    }
 
+   ////////////////////////////////////////////////////////////////////////////
+   /// \brief Fill and return a GraphAsymmErrors (*lazy action*).
+   /// \param[in] v1Name The name of the column that will fill the x axis.
+   /// \param[in] v2Name The name of the column that will fill the y axis.
+   /// \param[in] eXLName The name of the column of X low errors
+   /// \param[in] eXHName The name of the column of X high errors
+   /// \param[in] eYLName The name of the column of Y low errors
+   /// \param[in] eYHName The name of the column of Y high errors
+   /// \return the GraphAsymmErrors wrapped in a RResultPtr.
+   ///
+   /// Columns can be of a container type (e.g. std::vector<double>), in which case the graph
+   /// is filled with each one of the elements of the container.
+   /// If Multithreading is enabled, the order in which points are inserted is undefined.
+   ///
+   /// This action is *lazy*: upon invocation of this method the calculation is
+   /// booked but not executed. Also see RResultPtr.
+   ///
+   /// ### Example usage:
+   /// ~~~{.cpp}
+   /// // Deduce column types (this invocation needs jitting internally)
+   /// auto myGAE1 = myDf.GraphAsymmErrors("xValues", "yValues", "exl", "exh", "eyl", "eyh");
+   /// // Explicit column types
+   /// using f = std::vector<float>;
+   /// auto myGAE2 = myDf.GraphAsymmErrors<f, f, f, f, f, f>("xValues", "yValues", "exl", "exh", "eyl", "eyh");
+   /// ~~~
+   ///
+   /// \note Differently from other ROOT interfaces, the returned graph is not associated to gDirectory
+   /// and the caller is responsible for its lifetime (in particular, a typical source of confusion is that
+   /// if result histograms go out of scope before the end of the program, ROOT might display a blank canvas).
+   template <typename V1 = RDFDetail::RInferredType, typename V2 = RDFDetail::RInferredType,
+             typename EXL = RDFDetail::RInferredType, typename EXH = RDFDetail::RInferredType,
+             typename EYL = RDFDetail::RInferredType, typename EYH = RDFDetail::RInferredType>
+   RResultPtr<::TGraphAsymmErrors>
+   GraphAsymmErrors(std::string_view v1Name = "", std::string_view v2Name = "", std::string_view eXLName = "",
+                    std::string_view eXHName = "", std::string_view eYLName = "", std::string_view eYHName = "")
```

Comment:
Better to follow the same naming convention as https://root.cern/doc/master/classTGraphAsymmErrors.html#a7bb7122d8b07cd41851450c639bdee6a : 

```suggestion
   GraphAsymmErrors(std::string_view x = "", std::string_view y = "", std::string_view exl = "",
                    std::string_view exh = "", std::string_view eyl = "", std::string_view eyh = "")
```